security/rage/Portfile

```diff
@@ -0,0 +1,440 @@
+    if {[variant_isset mount]} {
+        xinstall -m 0755 ${cargo_builddir}/rage-mount ${destroot}${prefix}/bin/
+    }
+
+    # Install shell completions
+    xinstall -d -m 0755 ${destroot}${prefix}/share/bash-completion/completions
+    xinstall -m 0644 ${cargo_builddir}/completions/rage.bash \
+        ${destroot}${prefix}/share/bash-completion/completions/rage
+    xinstall -m 0644 ${cargo_builddir}/completions/rage-keygen.bash \
+        ${destroot}${prefix}/share/bash-completion/completions/rage-keygen
+    if {[variant_isset mount]} {
+        xinstall -m 0644 ${cargo_builddir}/completions/rage-mount.bash \
+            ${destroot}${prefix}/share/bash-completion/completions/rage-mount
+    }
+
+    xinstall -d -m 0755 ${destroot}${prefix}/share/fish/vendor_completions.d
+    xinstall -m 0644 ${cargo_builddir}/completions/rage.fish \
+        ${destroot}${prefix}/share/fish/vendor_completions.d/
+    xinstall -m 0644 ${cargo_builddir}/completions/rage-keygen.fish \
+        ${destroot}${prefix}/share/fish/vendor_completions.d/
+    if {[variant_isset mount]} {
+        xinstall -m 0644 ${cargo_builddir}/completions/rage-mount.fish \
+            ${destroot}${prefix}/share/fish/vendor_completions.d/
+    }
+
+    xinstall -d -m 0755 ${destroot}${prefix}/share/zsh/site-functions
+    xinstall -m 0644 ${cargo_builddir}/completions/_rage \
+        ${destroot}${prefix}/share/zsh/site-functions/
+    xinstall -m 0644 ${cargo_builddir}/completions/_rage-keygen \
+        ${destroot}${prefix}/share/zsh/site-functions/
+    if {[variant_isset mount]} {
+        xinstall -m 0644 ${cargo_builddir}/completions/_rage-mount \
+            ${destroot}${prefix}/share/zsh/site-functions/
+    }
+
+    # Install man pages
+    xinstall -d -m 0755 ${destroot}${prefix}/share/man/man1
+    xinstall -m 0644 {*}[glob ${cargo_builddir}/manpages/man1/*] \
+        ${destroot}${prefix}/share/man/man1/
+
+    xinstall -d -m 0755 ${destroot}${prefix}/share/man/es_AR/man1
+    xinstall -m 0644 {*}[glob ${cargo_builddir}/manpages/es_AR/man1/*] \
+        ${destroot}${prefix}/share/man/es_AR/man1/
+
+    xinstall -d -m 0755 ${destroot}${prefix}/share/man/it/man1
+    xinstall -m 0644 {*}[glob ${cargo_builddir}/manpages/it/man1/*] \
+        ${destroot}${prefix}/share/man/it/man1/
+
+    xinstall -d -m 0755 ${destroot}${prefix}/share/man/ru/man1
+    xinstall -m 0644 {*}[glob ${cargo_builddir}/manpages/ru/man1/*] \
+        ${destroot}${prefix}/share/man/ru/man1/
+
+    xinstall -d -m 0755 ${destroot}${prefix}/share/man/zh_CN/man1
+    xinstall -m 0644 {*}[glob ${cargo_builddir}/manpages/zh_CN/man1/*] \
+        ${destroot}${prefix}/share/man/zh_CN/man1/
+
+    xinstall -d -m 0755 ${destroot}${prefix}/share/man/zh_TW/man1
+    xinstall -m 0644 {*}[glob ${cargo_builddir}/manpages/zh_TW/man1/*] \
+        ${destroot}${prefix}/share/man/zh_TW/man1/
+}
+
+variant mount description {Build with pcre2} {
```

Comment:
The description for this variant needs to be corrected.  Since it seems like this `mount` variant modifies `rage` to build against `macfuse`, it seems like the description should instead say `Build with FUSE/MacFUSE`

Reply:
Fixed this one as well